src/vmem.c

```diff
@@ -211,6 +211,10 @@ static int vmem_read(void) {
       char *inst = key + strlen("pgsteal_");
       value_t value = {.derive = counter};
       submit_one(inst, "vmpage_action", "steal", value);
+    } else if (strncmp("pgscan_", key, strlen("pgscan_")) == 0) {
```

Comment:
This will hide the later checks for `pgscan_kswapd_` and `pgscan_direct_`. Imho it should be placed after those two.